Cargo.toml

```diff
@@ -47,6 +47,9 @@ httpmock = "0.7.0"
 predicates = "2.1"
 mockall = "0.12.1"
 
+[package.metadata.binstall]
+bin = "c2patool"
```

Comment:
@dyro I tried to add this in here since by default it tries to install the `signer-path-*` binaries that are used for testing (`cargo install` does this as well). Is there a way we can only build these for running tests?